cfme/tests/v2v/test_rest_migrations.py

```diff
@@ -189,3 +191,39 @@ def _cleanup():
     results = appliance.rest_api.response.json()["results"]
     assert results[0]["success"] is False
     assert results[1]["success"] is True
+
+
+@pytest.mark.parametrize("transformation_method", ["SSH", "VDDK"], ids=["SSH", "VDDK"])
+def test_rest_conversion_host_crud(appliance, source_provider, provider, transformation_method):
+    """
+    Tests conversion host crud via REST
+    Polarion:
+        assignee: ytale
+        casecomponent: V2V
+        caseimportance: high
+        testtype: functional
+        initialEstimate: 1/2h
+        startsin: 5.9
+        tags: V2V
+    """
+    # Test1: Create two conversion host
+    set_conversion_host_api(appliance, transformation_method, source_provider, provider)
+    conv_host = appliance.rest_api.collections.conversion_hosts.all
```

Comment:
Optional : Since you're going to be using `appliance.rest_api.collections.conversion_hosts` plenty of times in the test, you could store it in a variable and then use.

Reply:
So here `conv_host` variable holds static value, if I want to re-assigned conversation host count I have to use that again and again. We also have option to write small fixture with function scope but I am not considering it just for two repetition.

Reply:
I read it differently, thanks for offline clarification, just updated